src/main/java/io/moderne/organizations/OrganizationStructureService.java

```diff
@@ -53,11 +51,11 @@ public Map<String, OrganizationRepositories> readOrganizationStructure() {
                         String branch = fields[1].trim();
                         RepositoryInput repository = repositoryMapper.determineRepository(cloneUrl, branch);
                         if (repository == null) {
-                            if (allowMissingScmOrigins) {
+                            if (scmConfiguration.isAllowMissingScmOrigins()) {
                                 log.warn("No scm origin found for %s. Consider adding it to scm-origins.txt".formatted(cloneUrl));
                                 return;
                             } else {
-                                throw new IllegalStateException("No scm origin found for %s. Add it to scm-origins.txt or set organizations.allow-missing-scm-origins to true".formatted(cloneUrl));
+                                throw new IllegalStateException("No scm origin found for %s. Add it to moderne.scm.repositories or set moderne.scm.allow-missing-scm-origins to true".formatted(cloneUrl));
```

Comment:
Should we add `in application.yml` to this message so they know where to set it?